Content.Server/Chat/ChatManager.cs

```diff
@@ -93,6 +95,17 @@ public void SendOOC(IPlayerSession player, string message)
             _mommiLink.SendOOCMessage(player.SessionId.ToString(), message);
         }
 
+        public void SendDeadChat(IPlayerSession player, string message)
+        {
+            var clients = _playerManager.GetPlayersBy(x => x.AttachedEntity != null && x.AttachedEntity.HasComponent<GhostComponent>()).Select(p => p.ConnectedClient);;
+
+            var msg = _netManager.CreateNetMessage<MsgChatMessage>();
```

Comment:
No speech bubbles?

Reply:
Whoops, I completely forgot we had fancy speech bubbles when coding this--
Added them now.